drivers/espi/espi_npcx.c

```diff
@@ -872,8 +872,11 @@ static int espi_npcx_receive_vwire(const struct device *dev,
 
 			val = GET_FIELD(inst->VWEVMS[reg_idx],
 							NPCX_VWEVMS_WIRE);
+
+#if defined(CONFIG_ESPI_VWIRE_VALID_BIT_CHECK)
```

Comment:
Likewise IS_ENABLED() is preferred for simple tests that don't involve e.g. symbols that don't exist in certain conditions.

Reply:
done